bdd-generator/src/main/java/com/epam/test_generator/services/EmailServiceImpl.java

```diff
@@ -38,12 +38,11 @@
     /**
      * Sends message to register user
      * @param user entity of user who will be registered
-     * @param request
      */
     @Override
-    public UserDTO sendRegistrationMessage(User user, HttpServletRequest request) {
+    public UserDTO sendRegistrationMessage(User user, UriComponentsBuilder uriComponentsBuilder) {
         Token userConformationToken = tokenService.createToken(user, CONFIRMATION_TIME);
-        String confirmUrl = passwordService.createConfirmUrl(request, userConformationToken);
+        String confirmUrl = passwordService.createConfirmUrl(uriComponentsBuilder, userConformationToken);
         String subject = environment.getProperty("subject.registration.message");
         String text = environment.getProperty("registration.message");
         String site = environment.getProperty("site.name");
```

Comment:
I believe we can even use the uriBuilder for site name generateion (instead of environment.getProperty("site.name"))

Reply:
@hdghg Done.